SwiftGit2/SwiftGit2.m

```diff
@@ -34,3 +34,21 @@ git_checkout_options SG2CheckoutOptions(SG2CheckoutProgressBlock progress) {
 	
 	return result;
 }
+
+git_clone_options SG2CloneOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_clone_options result = GIT_CLONE_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
+}
+
+git_fetch_options SG2FetchOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_fetch_options result = GIT_FETCH_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
```

Comment:
It's not that I want them to be in Objective-C. It's that I want them to (1) be consistent and (2) encapsulate the manipulation of the git fields. #60 moves `SG2CheckoutOptions` into Swift, so this PR can do the same.